en/advanced_config/tuning_the_ecl_ekf.md

```diff
@@ -191,11 +191,26 @@ Optical flow data will be used if the following conditions are met:
 
 ### External Vision System
 
-Position and Pose Measurements from an external vision system, e.g. Vicon, can be used:
+Position, Velocity or Orientation Measurements from an external vision system, e.g. Vicon, can be used:
```

Comment:
Pose is orientation. Either way, can you lower case velocity and orientation.

Reply:
Pose is the combination of position and orientation. https://en.wikipedia.org/wiki/Pose_(computer_vision)